### ISSUE TITLE
Fix Request: false positive fix - JAXBContextCreatedForEachMethodCall, in a @PostConstruct, should be allowed 

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#170]
A @PostConstruct method is meant to be called only once.

```
import javax.annotation.PostConstruct;
import javax.xml.bind.JAXBContext;
import javax.xml.bind.JAXBException;

class Klass {
    private JAXBContext jaxbDocument;

    @PostConstruct
    protected void init() {
        try {
            jaxbDocument = JAXBContext.newInstance(Document.class);
        } catch (JAXBException e) { throw new IllegalArgumentException(e); }
    }
}
```
